spec/services/load_test_data_remover_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
+
+RSpec.describe LoadTestDataRemover do
+  let(:contact_email_address) { 'visitor@test.example.com' }
+  let(:prison_email_address) { 'prison@test.example.com' }
+  let(:prison) {
+    create(
+      :prison,
+      name: 'Leeds',
+      email_address: prison_email_address,
+      estate: create(:estate, nomis_id: 'LEI')
+    )
+  }
+  let(:prisoner) {
+    create(
+      :prisoner,
+      number: prisoner_number,
+      first_name: 'Load',
+      last_name: 'Test',
+      date_of_birth: Date.parse(prisoner_dob)
+    )
+  }
+
+  let(:vst) {
+    create(
+      :visit,
+      prison: prison,
+      contact_email_address: contact_email_address,
+      prisoner: prisoner
+    )
+  }
+  let(:prisoner_number) { 'A1475AE' }
+  let(:prisoner_dob) { '1979-04-23' }
+  let(:visitor) { vst.visitors.last }
+
+  context 'when visit requests are generated by load tests' do
+    let(:first_name) { 'Load' }
+    let(:last_name)  { 'Test' }
+
+    context 'with "remove_load_test_data" flag switched on' do
+      it 'removes them' do
+        remove_load_test_data(:on)
```

Comment:
We've already something similar you can use to stub the value of te configuration

https://github.com/ministryofjustice/prison-visits-2/blob/master/spec/support/helpers/configuration_helpers.rb